### PR TITLE
[W08H03] revert test parallel iterators

### PR DESCRIPTION
**Pull request by felixhauptmann**
Multiple iterator support (added in #168) is not necessary:
https://zulip.in.tum.de/#narrow/stream/1499-PGdP-W08H03/topic/mehrere.20iteratoren

### CODE REVIEW

File: w08h03/test/pgdp/datastructures/UnitTest.java [hunk 1]
```diff
@@ -155,7 +130,7 @@ public void hasNextTest() {
 		for (int j = 0; j <= i; j++) {
 			treeIt.next();
 			if(j == i) {
-				assertFalse(treeIt.hasNext());
+				Assertions.assertFalse(treeIt.hasNext());
```
Comment (Anatoly03) [inline w08h03/test/pgdp/datastructures/UnitTest.java:133]: Why?

Comment (felixhauptmann) [reply w08h03/test/pgdp/datastructures/UnitTest.java:133]: Ask the original author... Ill fix it

**Review by Anatoly03 — APPROVED** [review]

**Comment by Anatoly03** [comment]
🐧

**Review by nilsreichardt — APPROVED** [review]
LGTM

**Review by MaximilianAnzinger — COMMENTED** [review]
🐧

**Pull request merged**